data/changelog.txt

```diff
@@ -1,3 +1,630 @@
+# 0.G (Gernsback or Gaiman are my suggestions/guess)
+
+## Highlights
+Effect on condition brings a scripting pseudo language to Cataclysm.  Some of its key uses in 0.G have been the introduction of portal storms, NPC faction development over time and random encounters in already explored maps.
+The introduction of hobbies has added a new tab to character creation that allows the player to select one or more hobbies/backgrounds that flesh out what your character was doing prior to the Cataclysm in skills and traits.
+JSONized sidebar allows almost total control of what your sidebar looks like and what information it displays is up to you!
+Skill rust has been turned into a net gain for the player as any rusted xp now turns into an xp multiplier that helps your character learn faster.  As a result it now defaults to on.
```

Comment:
```suggestion
The skill system has been greatly expanded with many new proficiencies and many new ways to learn them, as well as the groundwork for practice-based skill improvement, including a skill rust system that doesn't suck because it caps effective skill lost and regenerates faster than regular skill gain.
```